File: pgm/ABC_Visual_Basic/ABC_161_A.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim X, Y, Z As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        X = arr(0) : Y = arr(1) : Z = arr(2)
        Console.WriteLine(Cstr(Z) + " " + CStr(X) + " " + CStr(Y))
    End Sub
End Class

File: pgm/ABC_C#/ABC_161_A.cs
using System;
// using System.Collections.Generic;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int X, Y, Z;
        X = int.Parse(arr[0]);
        Y = int.Parse(arr[1]);
        Z = int.Parse(arr[2]);
        Console.WriteLine(
            Z.ToString() + " " + X.ToString() + " " + Y.ToString());
    }
}